apps/webapp/src/state/send.ts

```diff
@@ -128,12 +128,20 @@ const planWitnessBuildBroadcast = async ({
   const { data: authorizationData } = await custodyClient.authorize({ plan });
   if (!authorizationData) throw new Error('no authorization data in response');
 
+  // Start timer
+  const startTime = performance.now(); // Record start time
+
```

Comment:
Can remove these timers

Reply:
accidentally committed! I already removed it locally